### PR TITLE
feat: add e2e tests

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to:**
https://oat-sa.atlassian.net/browse/AUT-122

**Description of changes**:
Sample E2E tests for item creation, edition, deletion.

**How to run locally:**
-  Checkout `feature/AUT-122/e2e` branch on [tao](https://github.com/oat-sa/tao-core/pull/2814)
- cd tao/views
- `npm install`
- rename `envs/env.sample` to `env.json` and update it with necessary config data
- `npm run cy:open` - to run in browser
- or `npm run cy:run` -  for headless execution

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
LGTM

**Review by bziondik — APPROVED** [review]
- [x]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/cypress/tests/item.spec.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    /**
+     * Delete newly created items after each step
+     */
+    afterEach(() => {
+        cy.deleteClassFromRoot(
+            selectors.root,
+            selectors.itemClassForm,
+            selectors.deleteClass,
+            selectors.deleteConfirm,
+            newClassName
```
Comment (jsconan) [inline views/cypress/tests/item.spec.js:48]: **thought:** I wonder if this is needed, Unless it is part of the test scenario, but in this case it should take place inside it. Usually, e2e test should not perform a cleanup internally.

**Review by jsconan — COMMENTED** [review]
It looks ok globally, except for the internal cleanup, but the test fails on my local instance
image

**Review by jsconan — APPROVED** [review]
It works well!

**Comment by jsconan** [comment]
**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**